analyzers/src/SonarAnalyzer.Common/Rules/Utilities/UtilityAnalyzerBase.cs

```diff
@@ -50,22 +51,22 @@ public abstract class UtilityAnalyzerBase : SonarDiagnosticAnalyzer
                 EndOffset = lineSpan.EndLinePosition.Character
             };
 
-        protected void ReadParameters(SonarAnalysisContext context, CompilationAnalysisContext c)
```

Comment:
We should consider setting `EnableConcurrentExecution` to `true` in line 40

https://github.com/SonarSource/sonar-dotnet/blob/53532700ba0c1d7e74490005423912067582c238/analyzers/src/SonarAnalyzer.Common/Rules/Utilities/UtilityAnalyzerBase.cs#L40

Reply:
You can create an issue to investigate if that will bring performance gain